src/KeyVault/KeyVault/Commands/NewAzureKeyVault.cs

```diff
@@ -117,76 +120,126 @@ public class NewAzureKeyVault : KeyVaultManagementCmdletBase
         [Parameter(Mandatory = false, HelpMessage = "Specifies the network rule set of the vault. It governs the accessibility of the key vault from specific network locations. Created by `New-AzKeyVaultNetworkRuleSetObject`.")]
         public PSKeyVaultNetworkRuleSet NetworkRuleSet { get; set; }
 
-        #endregion
+        /// <summary>
+        /// ApiVersion
+        /// </summary>
+        [Parameter(Mandatory = false,
+                   HelpMessage = "Specifies the SDK api version. Default value is '2023-02-01'.")]
+        public string ApiVersion { get; set; } = "2023-02-01";
 
-        public override void ExecuteCmdlet()
+        /// <summary>
+        /// FailOnExist
+        /// </summary>
+        [Parameter(Mandatory = false,
+                   HelpMessage = "Specifies whether the key vault will be created when existed. Default value: true. If set yes, the key vault will be overwritten incrementally.")]
+        public bool FailOnExist { get; set; } = true;
```

Comment:
```suggestion
        public switch FailOnExist { get; set; } = true;
```

Reply:
Better not to change, because the default value is true